lib/ecto/adapters/sqlite3/codec.ex

```diff
@@ -98,12 +98,31 @@ defmodule Ecto.Adapters.SQLite3.Codec do
     {:ok, value}
   end
 
-  # Ecto does check this already, so there should be no need to handle errors
-  def utc_datetime_encode(%{time_zone: "Etc/UTC"} = value) do
+  @text_datetime_format "%Y-%m-%d %H:%M:%S"
+
+  def utc_datetime_encode(%{time_zone: "Etc/UTC"} = value, :iso8601) do
     {:ok, NaiveDateTime.to_iso8601(value)}
   end
 
-  def naive_datetime_encode(value) do
+  def utc_datetime_encode(%{time_zone: "Etc/UTC"} = value, :text_datetime) do
+    {:ok, Calendar.strftime(value, @text_datetime_format)}
+  end
+
+  def utc_datetime_encode(%{time_zone: "Etc/UTC"}, type) do
+    raise ArgumentError,
+          "expected datetime type to be either `:iso8601` or `:text_datetime`, but received #{inspect(type)}"
+  end
+
+  def naive_datetime_encode(value, :iso8601) do
     {:ok, NaiveDateTime.to_iso8601(value)}
   end
+
+  def naive_datetime_encode(value, :text_datetime) do
+    {:ok, Calendar.strftime(value, @text_datetime_format)}
```

Comment:
I think this function is only available since elixir 1.11. The CI doesn't go below that but want to make sure you're not supporting any lower versions.

Reply:
I think it's reasonable to drop support for 1.11 when 1.14 is out of RC. My plan is to support at least the latest 3 minor versions.

Reply:
Oh that's perfect then since 1.10 and below are the problem for this particular function.

Reply:
Yea CI only runs for 1.11 up to 1.13

We should bump https://github.com/elixir-sqlite/ecto_sqlite3/blob/main/mix.exs#L10 to `~> 1.11`